lib/scrabble.rb

```diff
@@ -1,7 +1,23 @@
 class Scrabble
 
   def score(word)
-    1
+    return 0 if word.nil?
+
+    ones = [1] * word.length
+    score_with_multipliers(word, ones)
+  end
+
+  def score_with_multipliers(word, letter_multipliers, word_multiplier = 1)
+    total = 0
+    return total if word.nil?
+
+    word.each_char do |char|
+      letter_multiplier = letter_multipliers[word.index(char)]
```

Comment:
This is another case where your strategy is okay, but it is a bit dense to read. It may have been more readable if you saved word.index(char) to a variable, then used that variable to access the multiplier i.e. letter_multiplier[variable]